src/NotInstantiableWithoutWidgetFactoryInitializationException.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Widget;
+
+use Throwable;
+use Yiisoft\Definitions\Exception\NotInstantiableException;
+use Yiisoft\FriendlyException\FriendlyExceptionInterface;
+
+final class NotInstantiableWithoutWidgetFactoryInitializationException extends NotInstantiableException implements FriendlyExceptionInterface
+{
+    public function __construct(private Throwable $previous)
+    {
+        parent::__construct(
+            $previous->getMessage() .
+            ' Perhaps need initialize "WidgetFactory" with container for resolve dependencies.',
```

Comment:
- need to
- missed object: "you" or "widget"
- widget factory could be replace with full class name
- for -> to

Perhaps you need to initialize the factory with DI container to resolve dependencies 
Perhaps the factory should be initialized with DI container to resolve dependencies

Reply:
I'm used first way. Example of result:

`
Can not instantiate Yiisoft\Widget\Tests\Stubs\Injectable. Perhaps you need to initialize "Yiisoft\Widget\WidgetFactory" with DI container to resolve dependencies.
`